libs/akita-ng-effects/src/__tests__/actions.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { Actions } from '@datorama/akita-ng-effects';
+import { fakeAsync, flush } from '@angular/core/testing';
+import { Subject } from 'rxjs';
+
+describe('Actions', () => {
```

Comment:
https://stackblitz.com/edit/angular-testing-qknhej?file=app/hello.component.spec.ts

Reply:
I did the same thing in webstorm and it failed.
I'll fix it

Reply:
The tests won't pass correctly using the synchronous approach.

<img width="1162" alt="Bildschirmfoto 2020-12-20 um 18 40 41" src="https://user-images.githubusercontent.com/61085055/102720184-275fad80-42f3-11eb-9fea-c96dfb930da7.png">

Reply:
It doesn't make sense to me. I'll check it after the merge.

Reply:
Which font are you using?

Reply:
Should the next function be invoked before subscribing and asserting?

Reply:
I created that monospaced font myself.